wlhosted/templates/legal/documents/privacy.html

```diff
@@ -59,19 +59,19 @@ <h2>{% blocktrans %}Access to the Personal Data{% endblocktrans %}</h2>
 
 <h2>{% blocktrans %}The Personal Data retention{% endblocktrans %}</h2>
 
-<p>{% blocktrans %}The Personal Data is stored in the Service until User deletes his account in the service.{% endblocktrans %}</p>
+<p>{% blocktrans %}The Personal Data is stored in the Service until the User deletes their account on the service.{% endblocktrans %}</p>
 
-<p>{% blocktrans %}Access log information might be collected for longer period for the purpose of establishing, exercise or defence of legal claims.{% endblocktrans %}</p>
+<p>{% blocktrans %}Access log information might be collected for a longer period for the purpose of establishing, exercising or defending legal claims.{% endblocktrans %}</p>
 
 <h2>{% blocktrans %}Your rights{% endblocktrans %}</h2>
 
-<p>{% blocktrans %}User provides use Personal Data voluntarily. Without this we are not able to provide our services.{% endblocktrans %}</p>
+<p>{% blocktrans %}The User provides use of Personal Data voluntarily. Without this Weblate is not able to provide our services.{% endblocktrans %}</p>
```

Comment:
Either "The user provides User Personal Data", or "of".
```suggestion
<p>{% blocktrans %}The User provides use of Personal Data voluntarily. Without this Personal Data Weblate is not able to provide our services.{% endblocktrans %}</p>
```